### DISCUSSION TITLE
Support for cucumber upstream?

### DISCUSSION THREAD

**Discussion by blaisep** [pytest-dev/pytest-bdd#645] [Ideas]
I'm a contributor cucumber project and I would like to see pytest-bdd get all the latest goodness as it gets released upstream. We seem to be getting pretty close to pushing python bindings that would make this happen, but there's a lack of python expertise and a greater lack of familiarity with pytest-bdd

**Comment by jsa34** [comment]
I hope we are now on this with the v8.x releases!

The main focus of these releases have been about Cucumber compliance.
